live/live-root/usr/lib/dracut/modules.d/99agama-cmdline/save-agama-conf.sh

```diff
@@ -7,3 +7,15 @@
 if [ -e /etc/hostname ]; then
   cp /etc/hostname "$NEWROOT/etc/hostname"
 fi
+
+if [ -e /run/agama/copy_network ]; then
+  if getargbool 1 inst.copy_network; then
+    mkdir -p /run/NetworkManager/conf.d
+    echo '[main]' >/run/NetworkManager/conf.d/00-agama-server.conf
```

Comment:
change looks good, but I would like to see here documentation *WHY* we create those entries and why we copy all system connections.